action-core/src/main/java/com/adyen/checkout/action/core/internal/ActionHandlingPaymentMethodConfigurationBuilder.kt

```diff
@@ -22,7 +22,11 @@ import com.adyen.checkout.voucher.VoucherConfiguration
 import com.adyen.checkout.wechatpay.WeChatPayActionConfiguration
 import java.util.Locale
 
-@Suppress("UNCHECKED_CAST")
+@Suppress(
+    "UNCHECKED_CAST",
+    "ktlint:standard:discouraged-comment-location",
+    "ktlint:standard:type-parameter-list-spacing",
```

Comment:
What do you think of modifying the constructor declaration and moving the comment outside of the class? E.g.
```
/**
 * Initialize a configuration builder with the required fields.
 *
 * @param shopperLocale The [Locale] of the shopper.
 * @param environment The [Environment] to be used for internal network calls from the SDK to Adyen.
 * @param clientKey Your Client Key used for internal network calls from the SDK to Adyen.
 */
abstract class ActionHandlingPaymentMethodConfigurationBuilder<
    ConfigurationT : Configuration,
    BuilderT : BaseConfigurationBuilder<ConfigurationT, BuilderT>
    >(
    shopperLocale: Locale,
    environment: Environment,
    clientKey: String
) : BaseConfigurationBuilder<ConfigurationT, BuilderT>(shopperLocale, environment, clientKey),
    ActionHandlingConfigurationBuilder<BuilderT> {
...
```
Though I am not sure which one is better, since the way I sent above, the code might be less readable, but that way we can remove both of the Suppressed warnings. 😶‍🌫️

Same applies to the rest of the Suppressed `ktlint` warnings in this PR.

Reply:
The main issue with that approach is that the comment appears on the class and not on the constructor. So if you hover over this class it tells you about something that is not directly relevant